lab-ben/lib/storage.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const debug = require('debug')('http:storage');
+const storage = {};
+const Promise = require('bluebird');
+const fs = Promise.promisifyAll(require('fs'), {suffix: 'Prom'});
+
+module.exports = exports = {};
+
+exports.createItem = function(folder, item) {
+  debug('#createItem');
+
+  if(!folder) return Promise.reject(new Error('schema required'));
+  if (!item) return Promise.reject(new Error('item required'));
+
+  console.log(item);
+
+  fs.writeFileProm(`./data/${folder}/${item.id}.txt`, JSON.stringify(item))
+  .then(Promise.resolve(item));
+
+
+
+  // console.log(item, 'storage log');
+
```

Comment:
**TODO FOR LOST POINTS**

Lab 9 requires file persistence. See me if you need help getting that done.